src/wstool/config_yaml.py

```diff
@@ -355,7 +355,10 @@ def get_path_spec_from_yaml(yaml_dict):
             elif key == "uri":
                 uri = value
             elif key == "version":
-                version = value
+                if value is not None: 
+                    version = str(value)
```

Comment:
might add a comment here explaining that int values cause later

Reply:
Added comment